src/debug/HyprNotificationOverlay.cpp

```diff
@@ -1,6 +1,20 @@
+#include <numeric>
+#include <pango/pangocairo.h>
 #include "HyprNotificationOverlay.hpp"
 #include "../Compositor.hpp"
-#include <pango/pangocairo.h>
+#include "../config/ConfigValue.hpp"
+#include "../helpers/FontUtils.hpp"
+
+eIconBackend get_iconBackendID(const std::string& fontFamily) {
```

Comment:
what the fuck is this naming convention lmao

iconBackendFromFont

Reply:
🤒 my fault